backend/tournaments/tools.py

```diff
@@ -72,35 +72,66 @@ def get_tournament_buckets(tournament):
 
   Return:
     List of dictionaries [{'name': <tournament name>,
-                           'members': [<username>, ...]}, ...]
+                           'members': [{
+                              'name': <player name>,
+                              'flag': <player country code>,
+                              'rating': <fixed rating>
+                              }, ...]}, ...]
 
   Throws:
     NotFound -- if tournament is not found.
   """
+  ratings = {}
+  for player in get_tournament(tournament).players.all():
+    ratings[player.member.name()] = player.fixed_rating
   buckets = get_tournament(tournament).bucket_set.all()
   return [
       { 'name': b.name,
-        'members': [ m.member.name() for m in b.players.all() ] }
+        'members': [ 
+      { 'name': m.member.name(),
+        'flag': m.member.country,
+        'rating': ratings[m.member.name()] }
```

Comment:
Either the bot, or we can fetch it from www.ficsgames.org.

If this is the bot, it will only fetch the rating, after the user signs up the tournament on the webpage. So between the sign-up and bot rating fetch, the user will temporarily have no fixed rating.